extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -48,7 +48,6 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverDelayCountries';
 		$vars[] = 'wgAdDriverDelayTimeout';
 		$vars[] = 'wgAdDriverDisableSraCountries';
-		$vars[] = 'wgAdDriverDisableBadgeAdCountries';
```

Comment:
Removing variable from IG at this moment can enable Badge Ad on cached pages. Please bring it back here ;)

Reply:
That's true. Restored, creating new blocked task to remember to remove that after deploy.